msg/de_DE.utf8/utils.msg

```diff
@@ -490,7 +490,9 @@ gültige Optionen:\n\
   -d, --data-file=DATEI            DATEI laden\n\
   -t, --table=TABLE                Name der Tabelle, die für das fehlende Klassenheader in der Datei ersetzt wird\n\
       --error-control-file=DATEI   DATEI für Fehlerkontrolle während Ladung\n\
-      --ignore-class-file=DATEI    Eingangsdatei für Klassenamen, die nicht geladen werden\n
+      --ignore-class-file=DATEI    Eingangsdatei für Klassenamen, die nicht geladen werden\n\
+      --no-user-specified-name     Find class, serial, and trigger by object-name and don't use user-specified-name.\n
```

Comment:
How about below?
no-use-user-schema For backward compatibility with the load file of the previous version of V11.2.

Reply:
Yes, I think it should be included. I will add.